templates/infrastructure/environment/aws/maria_db/terragrunt.hcl

```diff
@@ -0,0 +1,65 @@
+include {
+  path = find_in_parent_folders()
+}
+
+locals {
+  region = local.region_vars.locals.aws_region
+}
+
+terraform {
+
+  # the below config is an example of what the config should like
+  source = "github.com/argonautdev/tf-modules.git//modules/aws/maria_db?ref={{.RefVersion}}"
+}
+
+dependency "vpc" {
+  config_path = "../vpc"
+   mock_outputs = {
+    vpc_id = "temporary-dummy-id",
+    public_subnets = ["temporary-dummy-subnet-1", "temporary-dummy-subnet-2"],
+    private_subnets = ["temporary-dummy-subnet-1", "temporary-dummy-subnet-2"],
+    database_subnets = ["temporary-dummy-subnet-1", "temporary-dummy-subnet-2"],
+    vpc_cidr_block = "10.0.0.0/8"
+    default_security_group_id = "default"
+  }
+}
+
+
+# These are the variables we have to pass in to use the module specified in the terragrunt configuration above
+inputs = {
+  default_tags = {
+    "argonaut.dev/name"        = "{{ .Spec.name }}"
+    "argonaut.dev/type"        = "RDS"
+    "argonaut.dev/manager"     = "argonaut.dev"
+    "argonaut.dev/rds-engine"  = "{{ .Spec.engine }}"
+    "argonaut.dev/env/${local.env}" = "true"
+  }
+  aws_region = "${local.region}"
+
+  visibility = "{{ .Spec.visibility }}"
+
+  instance_name     = "{{ .Spec.instance_name }}"
+  database_name           = "{{ .Spec.database_name }}"
+  multi_az = "{{.Spec.multi_az }}"
+  backup_retention_period = "{{.Spec.backup_retention_period }}"
+  engine_version = "{{ .Spec.engine_version}}"
```

Comment:
could you revert this back to the older version with a different condition maybe?